app/dashboard/management/commands/sync_listener.py

```diff
@@ -22,64 +22,107 @@
 from django.core.management.base import BaseCommand
 
 from dashboard.utils import (
-    get_bounty, get_web3, getBountyContract, getStandardBountiesContractAddresss, web3_process_bounty,
+    Web3, get_bounty, get_web3, getBountyContract, getStandardBountiesContractAddresss, web3_process_bounty,
 )
 
-logging.getLogger("requests").setLevel(logging.WARNING)
 logging.getLogger("urllib3").setLevel(logging.WARNING)
-
-
-def process_bounty(bounty_id, network):
-    bounty = get_bounty(bounty_id, network)
-    return web3_process_bounty(bounty)
+logging.getLogger("web3.RequestManager").setLevel(logging.WARNING)
+logging.getLogger("web3.providers.WebsocketProvider").setLevel(logging.WARNING)
+logging.getLogger("websockets").setLevel(logging.WARNING)
+
+
+EVENTS = [
+            'BountyIssued',
+            'BountyActivated',
+            'ContributionAdded',
+            'BountyFulfilled',
+            'FulfillmentAccepted',
+            'BountyChanged',
+            'BountyKilled',
+            'PayoutIncreased',
+            'FulfillmentUpdated',
+            'DeadlineExtended',
+            'IssuerTransferred'
+        ]
+
+
+def process_entry(entry, network):
+    event = entry.event
+    tx_hash = Web3.toHex(entry.transactionHash)
+    bounty_id = entry.args.get('bountyId') or entry.args.get('_bountyId')
+    subdomain = ''
+    if network != 'mainnet':
+        subdomain = f'{network}.'
+    print(f'{event} {bounty_id} https://{subdomain}etherscan.io/tx/{tx_hash}')
+    print(entry.args)
+
+    try:
+        bounty = get_bounty(bounty_id, network)
+        did_change, old_bounty, new_bounty = web3_process_bounty(bounty)
+        if event == 'BountyActivated':
+            new_bounty.tx_hash_activated = tx_hash
+            new_bounty.save()
+            return True
+        if event == 'FulfillmentAccepted':
+            new_bounty.tx_hash_accepted = tx_hash
+            new_bounty.save()
+            return True
```

Comment:
is it worth adding the other events to this list?

Reply:
@owocki 
Most of the time we have > Issued|Activated|ContributionAdded| > BountyFulfilled|BountyKilled > FulfillmentAccepted.

We can save all events and make those hash properties a list.

Reply:
> Most of the time 

sure, but what about IncreasePayout event?  and the newFulfillment event?  those are pretty common too.

Reply:
yes that makes sense to me